linkedList.js

```diff
@@ -3,6 +3,101 @@
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
+
 function linkedListGenerator(){
 
+  let head = null;
+  let tail = null;
+
+  function getHead(){
+    return head;
+  }
+
+  function getTail(){
+    return tail;
+  }
+  
+  function add(value){
+    if (!head){
+      head = {value:value, next:null};
+      tail = head;
+      return head;
+    }
+
+    let node = head;
+    while (node.next){
+      node = node.next;
+    }
+    node.next = {value:value, next:null};
```

Comment:
This is often written as:
```
head = { 
  value: value,
  next: null
}
```

Ideally, you could create this node at the very top of the function since you know you'll be using it at some point.